controller/space.go

```diff
@@ -376,6 +400,10 @@ func ConvertSpaceToModel(appSpace app.Space) space.Space {
 		appSpace.Relationships.OwnedBy.Data != nil && appSpace.Relationships.OwnedBy.Data.ID != nil {
 		modelSpace.OwnerId = *appSpace.Relationships.OwnedBy.Data.ID
 	}
+	if appSpace.Relationships != nil && appSpace.Relationships.SpaceTemplate != nil &&
```

Comment:
why do you need to verify all these non-nil values if the `validateCreateSpace` function already takes care about it ?